qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java

```diff
@@ -432,6 +432,31 @@ public void doesNotThrowExceptionIfImportsOnly() throws Exception {
     }
 
     /**
+     * Test catch parameter names.
+     * @throws Exception In case of error
+     */
+    @Test
+    public void testCatchParameterNames() throws Exception {
```

Comment:
@kitsook have a look at http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names